src/main/java/com/google/api/codegen/discovery/Document.java

```diff
@@ -66,23 +68,35 @@ public static Document from(DiscoveryNode root) {
     String servicePath = root.getString("servicePath");
     String title = root.getString("title");
     String version = root.getString("version");
+    String id = root.getString("id");
```

Comment:
Small nit, but I tried to order the assignment of fields roughly in the order that their functions are declared in the `Document`, so this would be assigned right after description.